src/utils/fetchAndCache.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Fetch and cache a request
+ * @param {string|URL} url
```

Comment:
Beauty!
How's this used? Are there some tools that can consume this info?

Reply:
So, this is called [JSDoc](https://jsdoc.app/index.html). In editors that support this (like VSCode), it serves as documentation/hinting as well as can be used for type checking with ts-check (like TypeScript or Flow).
![image](https://user-images.githubusercontent.com/8426945/67035480-5a2e7480-f137-11e9-8ab7-2acc2898ea0a.png)